rancher/installing-rancher/installing-server/basic-ssl-config/index.md

```diff
@@ -17,24 +17,21 @@ Besides the typical Rancher Server [requirements]({{site.baseurl}}/rancher/insta
 
 ## Launching Rancher Server
 
-In our configuration, all traffic will pass through the proxy and be sent over a Docker link to the Rancher Server container. There are alternative approaches that could be followed, but this approach is simple and translates well. 
+In our configuration, all traffic will pass through the proxy and be sent over a Docker link to the Rancher Server container. There are alternative approaches that could be followed, but this approach is simple and translates well.
 
 Start the Rancher Server container with the additional environment variables.
 
 ```bash
 $ sudo docker run -d --restart=always --name=rancher-server \
--e "CATTLE_API_ALLOW_CLIENT_OVERRIDE=true" \
--e "CATTLE_HOST_API_PROXY_SCHEME=wss" rancher/server
+-e "CATTLE_API_ALLOW_CLIENT_OVERRIDE=true" rancher/server
```

Comment:
Good catch @cloudnautique. I kept it in there thinking it was needed for for the client IP.
But, x-forwarded-for is respected by default, so that setting is only needed if you are setting the `X-API-client-ip` header, which this setup is not.

So, I'm going to remove it.